src/util/eventFramework/tags.js

```diff
@@ -57,4 +57,17 @@ const getTransferSpanTags = ({ payload, headers, params }, transactionType, tran
   }
 }
 
-module.exports = { getTransferSpanTags }
+const getSpanTags = (transactionType, transactionAction, transactionId, source, destination) => {
+  return {
+    transactionType,
+    transactionAction,
+    transactionId,
+    source,
+    destination
+  }
+}
```

Comment:
Do we really need this function? Unless there is processing, it makes no sense to me.

Reply:
its one generic way of creating tags for spans i believe it is needed. this will be used for all none transfer tags on all APIs

Reply:
Shouldn't this be part of the event-sdk?

I also agree that it's probably not needed.

Reply:
> Shouldn't this be part of the event-sdk?
> 
> I also agree that it's probably not needed.

The tags creation for transfer spans is here, should that then also not be in the event-sdk? 
If the bas tags are going to differ from this list then sure it should be removed, but if not then it is useful as the default tags are defined somewhere

Reply:
Fair enough, but @ggr's comment still stands...it's like having a function for the sake of a function with functionality that is offered out of the box :D

Reply:
Lol ok so @mdebarros and @ggrg what should i do?
move this into the event-sdk? or to the sdk-scheme-adapter? or keep it here?